### ISSUE TITLE
inventory moves in eln gui's when using nei

### ISSUE THREAD

**Issue by SirKastros** [Electrical-Age/ElectricalAge#96]
the player's inventory move down when you look something up in nei when the gui for an eln machine is open.

**Comment by JPitman97** [comment]
Yes this mod is currently not fully compatible with Not Enough Items but we hope to have it fully compatible in the not so distant future

**Comment by Barhandar** [comment]
ELN's UIs also completely override controls other than "Esc". Is fine for Project Red: Integration, very much not fine for tech-heavy mod where you would look up recipes and items all the time.
